### ISSUE TITLE
Prometheus Addon: embedded grafana instance makes an outbound request even in airgapped environments

### ISSUE THREAD

**Issue by dexhorthy** [replicatedhq/kURL#429]
It appears that by default, Grafana pings out to see if there's a newer version of Grafana available. I think this is a `grafana.ini` param, the helm/charts/stable version has a values.yaml parameter to control this:

```yaml
grafana.ini:
  paths:
    data: /var/lib/grafana/data
    logs: /var/log/grafana
    plugins: /var/lib/grafana/plugins
    provisioning: /etc/grafana/provisioning
 analytics:
   check_for_updates: true
```

we need to disable this in airgapped scenarios (should we just disable it across the board?)

**Comment by marccampbell** [comment]
I'm in favor of disabling this everywhere, or else we end up with 2 things managing the update lifecycle of this component, which could end up causing upgrades and downgrades constantly.

**Comment by dexhorthy** [comment]
👍 sounds good to me

**Comment by emosbaugh** [comment]
Resolved by #684
